core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -165,6 +171,9 @@ var PreferencesPage = function() {
   };
 
   this.expectUserBioToBe = async function(bio) {
+    waitFor.visibilityOf(
+      userBioElement,
+      'User bio field takes too long to appear.');
```

Comment:
Can we move this to the line above? (Following 80 chars ina line) [Ditto for the above cases]

Reply:
Done.